dova/www/js/controllers.js

```diff
@@ -290,6 +306,7 @@ function ($scope, $stateParams, $http, $rootScope, $state) {
 
 .controller('diagnosisCtrl', ['$scope', '$stateParams', '$http', '$rootScope', '$state',
 function ($scope, $stateParams, $http, $rootScope, $state) {
+$scope.offline_debug = false;
```

Comment:
I use find and replace so I won't forget about it, its just left over from other testing